lib/linkable.rb

```diff
@@ -0,0 +1,8 @@
+module Linkable
```

Comment:
Really, really mild nit, but I'm not sure about organizing a utility method like this? 

Naming wise, if I see 'include Linkable' in a class, I think that should be adding functionality to make that class linkable (by analogy with Hydra's Embargoable class, for instance), rather than just pulling in a utility function with that name.

It might be better to put this function in a file in lib/ (link_utils.rb or something), and then autoload it in application.rb:

```ruby
config.autoload_paths += Dir["#{config.root}/lib/**/"]
```

 This would also eliminate the need to require it in the application_helper.